src/main/java/org/example/model/Task.java

```diff
@@ -1,57 +1,13 @@
 package org.example.model;
 
-public class Task {
-    private int id;
-    private String title;
-    private String description;
-    private boolean isFinished;
+import org.immutables.value.Value;
 
-    public Task() {
+@Value.Immutable
```

Comment:
I'm curious, by attaching this "directive",does it makes the classname to prepend the Immutable keyword ?

Reply:
It generates a new class with Immutable infront of it with a bunch of helper methods https://immutables.github.io - it also doesn't all you to mutate on it